src/libraries/Microsoft.Extensions.Options/src/OptionsFactory.cs

```diff
@@ -94,10 +94,9 @@ public TOptions Create(string name)
         /// <summary>
         /// Creates a new instance of options type.
         /// </summary>
-        /// <param name="name">The name of the <typeparamref name="TOptions"/> instance to create.</param>
         /// <returns>The created <typeparamref name="TOptions"/> instance.</returns>
         /// <exception cref="MissingMethodException">The <typeparamref name="TOptions"/> does not have a public parameterless constructor or <typeparamref name="TOptions"/> is <see langword="abstract"/>.</exception>
-        protected virtual TOptions CreateInstance(string name)
```

Comment:
As #92549, changing this is not acceptable.

Reply:
This was brought from your existing `main` branch. It's not suggested to use `main` branch for contribution, open a new branch instead.